specification/compute/Compute.Management/models.tsp

```diff
@@ -1830,6 +1906,25 @@ model EventGridAndResourceGraph {
    * Specifies if event grid and resource graph is enabled for Scheduled event related configurations.
    */
   enable?: boolean;
+
+  /**
+   * Specifies the api-version to determine which Scheduled Events configuration schema version will be delivered.
+   */
+  @added(Versions.v2025_04_01)
+  scheduledEventsApiVersion?: string;
+}
+
+/**
+ * Specifies if Scheduled Events should be auto-approved when all instances are down.
+ */
+@added(Versions.v2025_04_01)
+model AllInstancesDown {
+  /**
+   * Specifies if Scheduled Events should be auto-approved when all instances are down.
+   * its default value is true
+   */
+  @added(Versions.v2025_04_01)
+  automaticallyApprove?: boolean;
```

Comment:
ARM recommends Enums over boolean

Reply:
In this case AutomaticallyApprove is already present in other definitions of a ScheduledEventsPolicy as a property of UserInitiatedRedeploy and UserInitiatedReboot and both of them are boolean. Changing this property to enum would break a breaking change so we would like an exemption on this.
https://github.com/Azure/azure-rest-api-specs/blob/4714ffda08ac079db3e4172f91b9810fd84c00ae/specification/compute/Compute.Management/models.tsp#L1890
https://github.com/Azure/azure-rest-api-specs/blob/4714ffda08ac079db3e4172f91b9810fd84c00ae/specification/compute/Compute.Management/models.tsp#L1880